inlang/source-code/sdk/src/createNodeishFsWithAbsolutePaths.ts

```diff
@@ -42,5 +42,11 @@ export const createNodeishFsWithAbsolutePaths = (args: {
 			path: string,
 			options: { signal: AbortSignal | undefined; recursive: boolean | undefined }
 		) => args.nodeishFs.watch(makeAbsolute(path), options),
+		// This might be surprising when symlinks were intended to be relative
```

Comment:
@martin-lysk i added implementations for the new methods which you had already added to the test. Could you check that they look ok.

Not 100% sure why we need this wrapper at all, but will focus on getting tests to pass for this PR.

Reply:
@jldec We need this wrapper because relative paths do not exist in non-server environments. 

Any path that is resolved by the SDK needs to be absolute and resolve from the project directory itself. Alternatively, we could set a fake CWD maybe

Reply:
thanks for clearing that up - @samuelstroschein